plugins/decklink/DecklinkInput.cpp

```diff
@@ -81,12 +73,19 @@ bool DeckLink::Activate(DeckLinkDevice *device, long long modeId)
 	if (instance == nullptr)
 		return false;
 
-	DeckLinkDeviceMode *mode = GetDevice()->FindMode(modeId);
+	if (curDevice == nullptr) {
+		LOG(LOG_ERROR, "Tried to activate an input without nullptr device.");
```

Comment:
I'm confused by this error message.  If curDevice is nullptr, the error says "without nullptr device"?